tests/NameCaseTest.php

```diff
@@ -22,20 +22,6 @@ final class NameCaseTest extends TestCase
         "Yusof bin Ishak",
     ];
 
-    private $macNames = [
```

Comment:
I will create new tests using those exceptions, thanks for highlighting the issue.